src/pybind/mgr/dashboard/frontend/src/app/app.component.scss

```diff
@@ -5,3 +5,28 @@
   margin: 0;
   padding: 0;
 }
+
+::ng-deep #toast-container {
+  z-index: 0;
```

Comment:
I think this is responsible for the render error.

Reply:
Fixed.